### PR TITLE
Standard assets

### PR DESCRIPTION
**Pull request by keldorg**
Replaced Space force assets for standard ones from MM
- Top Bar
- Lines
- Bet options

### CODE REVIEW

**Comment by keldorg** [comment]
retest this please

**Comment by keldorg** [comment]
retest this please

File: test/client/freeSpin.spec.rb [hunk 1]
```diff
@@ -72,9 +71,8 @@ def assert_free_spin (currentSpin, totalSpins)
             @scene.assert_alpha 0, "linesPanel.freeSpinsWin.textYouWon", 30
         end
 
-        it "hides the topBar marquee then shows the jackpot background" do
+        it "hides the topBar marquee" do
             @scene.assert_alpha 0, "topBar.marquee", 20
-            @scene.assert_alpha 1, "topBar.jackpotsBackground", 20
```
Comment (Nahikariii) [inline test/client/freeSpin.spec.rb]: why was this removed?

Comment (keldorg) [reply test/client/freeSpin.spec.rb]: The is no more this element in new instant-slots!

**Comment by Nahikariii** [comment]
I think you are missing a master build? https://jenkins2.gamevy.com/view/instant-slots/

**Comment by keldorg** [comment]
@Nahikariii Your are right, It is already done!

**Comment by keldorg** [comment]
retest this please
